docs/application/get-started/mobile.md

```diff
@@ -1,40 +1,49 @@
-# Get Started for Mobile
 
-The Get Started Tutorial teaches you how to create basic Tizen applications for mobile devices.
+# Get Started with Tizen Mobile Applications
+Samsung expands the Tizen ecosystem with advanced Tizen-based smart phone application development.  
 
-The Tizen supports three primary application types for mobile:
+## Mobile Native Applications
+You can create rich applications in C with device-specific features, advanced system settings, and additional functionalities.
 
--   [.NET application](#net-application) is a new way to develop applications for the Tizen operating system using C# language.
+[Quick start](../native/get-started/mobile/first-app.md) for beginners
```

Comment:
"Quick start for beginners" is used because in the design it is a button.
For more information, see
![Quickstart_button](https://user-images.githubusercontent.com/36875286/59590334-50d6c200-90db-11e9-9663-2ff1bc02d0b2.PNG)

If we are not using this design, then we can rephrase it to sentence format.